### DISCUSSION TITLE
Azure

### DISCUSSION THREAD

**Discussion by walterdanielnunez** [picocss/pico#535] [General]
The pico.azure.min.css or pico.azure.css is not included?

**Comment by rokf** [comment]
![CleanShot 2024-05-05 at 19 45 27](https://github.com/picocss/pico/assets/2836852/681118aa-f8b9-4d8f-8a62-8057bb61ffc6)

Azure is the default theme, it doesn't have `azure` in its name.

**You'd expect that to be included for consistency?**

**Reply by walterdanielnunez** [reply]
Oh, I didn't know azure was the default theme. I am new, using Pico CSS. Thanks for the answer!

**Comment by martinguenther** [comment]
I stumbled upon this, too, as I wanted to use specific nuances of the azure style. They are not fully included in the default theme. For example, there is no equivalent to `pico-color-azure-50` or `pico-background-azure-50`. Also others are missing.
To use these I would have to include the entire _pico.colors.min.css_ (which is obviously not recommended). Therefore it'd be great to have a separate _pico.azure.min.css_
